src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/DefaultRazorConfigurationService.cs

```diff
@@ -130,7 +130,7 @@ private void ExtractVSCodeOptions(
             if (parsedFormat.TryGetPropertyValue("enable", out var parsedEnableFormatting) &&
                 parsedEnableFormatting is not null)
             {
-                enableFormatting = GetObjectOrDefault(parsedEnableFormatting, enableFormatting);
+                formatting = GetObjectOrDefault(parsedEnableFormatting, formatting);
```

Comment:
Shouldn't this still be parsed as a `bool`, and then converted to `FormattingFlags.Enabled` as necessary? Or are you planning on changing the VS Code option?

Reply:
doi, yes. Just blanket chnaging things. Should still be a bool in vs code